Source/CarthageKit/Archive.swift

```diff
@@ -58,12 +58,12 @@ private func untargz(archive fileURL: URL, to destinationDirectoryURL: URL) -> S
 		.then(SignalProducer<(), CarthageError>.empty)
 }
 
-private let ArchiveTemplate = "carthage-archive.XXXXXX"
+private let archiveTemplate = "carthage-archive.XXXXXX"
```

Comment:
It'd definitely be nice to move them onto types instead.

The Carthage codebase is quite old for a Swift project—it was written in the early days of Swift 1. So it doesn't always follow conventions atm.